docs/API.md

```diff
@@ -39,7 +39,7 @@ Here's the full list of endpoints implemented so far. In all examples, replace e
 Responds with a simple "OK", signaling that the supervisor is alive and well.
 
 #### Examples:
-From an application container:
+From a service:
```

Comment:
I'm divided if this should be just `service` or using something `service container` is better. In my understanding, the service is the abstraction, but the container is the code actually running there. I'm not sure if that is the common view though

Reply:
`From an app container` works for me